lib/grpperm.gi

```diff
@@ -466,6 +466,48 @@ InstallMethod( NrMovedPoints,
     G -> NrMovedPoints( GeneratorsOfGroup( G ) ) );
 
 
+#############################################################################
+##
+#M  OrbitsMovedPoints( <G> )
+##
+InstallMethod(OrbitsMovedPoints,"for a permutation group",true,[IsPermGroup],0,
+function(G)
+local o;
+  o:=Orbits(G,MovedPoints(G));
+  o:=List(o,x->Immutable(Set(x)));
+  Sort(o);
+  return o;
+end);
+
+
+
+
+#############################################################################
+##
+#M  \=( <G>, <H> )  . .  . . . . . . . . .  test if two perm groups are equal
+##
+InstallMethod( \=, "perm groups", IsIdenticalObj, [ IsPermGroup, IsPermGroup ],
+function ( G, H )
+  # avoid new stabilizer chains if they are not computed yet
+  if OrbitsMovedPoints(G)<>OrbitsMovedPoints(H) then
+    return false;
+  fi;
+  if not (HasGeneratorsOfGroup(G) and HasGeneratorsOfGroup(H)) then
+    TryNextMethod();
+  fi;
+  if IsEqualSet( GeneratorsOfGroup( G ), GeneratorsOfGroup( H ) ) then return true;fi;
+
+  if Size(G)<>Size(H) then return false;fi;
```

Comment:
This probably wants to be `HasSize(G) and HasSize(H) and Size(G)<>Size(H)`, because otherwise GAP will just go and build stabilizer chains for both `G` and `H`, to find their size, which makes the next part pointless.

Reply:
This size comparison comes just before the backtrack. Presumably there should be an extra check with HasSize at the start. will change.

Reply:
I don't know if you left this line in accidentally when you moved the Size check to the top. I think it will be better to remove this and let the StabChain stuff, as that is designed to only require a stab chain (and therefore Size) on only one of the two groups.

Reply:
Done.